tests/actions/IOUTest.js

```diff
@@ -686,7 +686,7 @@ describe('actions/IOU', () => {
                                     waitForCollectionCallback: true,
                                     callback: (transaction) => {
                                         Onyx.disconnect(connectionID);
-                                        expect(transaction.pendingAction).toBe(CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
+                                        expect(transaction.pendingAction).toBeFalsy();
```

Comment:
Why the pending action changed to false instead of add?

Reply:
In this case, there are some error while creating transaction so we remove pendingAction in FailureData. It is reason why I update UTs here